bucket/imagemagick.json

```diff
@@ -19,7 +22,7 @@
         "conjure.exe",
         "convert.exe",
         "dcraw.exe",
-        "ffmpeg.exe",
+        ["ffmpeg.exe", "iffmpeg"],
```

Comment:
I'm happy to undo it, if it's an issue. We could solve this in one of three ways:

1. Add a "priority" flag to each app (or each exe), and assign a higher priority to ffmpeg. install/uninstall then needs to search *all* installed apps to see if *any* app supplies an exe. Ugh.
2. When a new app installs/updates over a shim,  ask the user which one s/he wants. Breaks the ffmpeg command, if the user selects the imagick version, but later uninstalls it. Yuk.
3. Add ffmpeg's path to PATH before the shims path. Yuk, but doesn't break on uninstall.